cmake/sysbuild/b0_mcuboot_signing.cmake

```diff
@@ -0,0 +1,168 @@
+# Copyright (c) 2020-2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+# This file includes extra build system logic that is enabled when
```

Comment:
Should we mention that this is a copy of of `zephyr/cmake/mcuboot.cmake` ?

Reply:
Not that I can see, many non-SB files have been ported to new files for SB functionality, they are not the same. And this file is not a copy of that file, it's actually a copy of a file from zephyr.